source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -15,3 +15,138 @@ Install with Confluent Cloud
+
+            1. Create global access.
+         
+            2. Create a service account with granular access.
+
+               If you create a service account, you must add the ``Operator`` role for the 
+               environment and the ``CloudClusterAdmin`` role to the cluster. 
+
+            Save these configurations to add to your ``user.properties`` file.
+
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__. 
+
+         .. step:: Configure Kafka cluster API keys
+
+            To access your Kafka cluster, you must configure your Kafka cluster API keys. 
+            Navigate to your :guilabel:`Cluster Overview` to create your Kafka cluster API 
+            keys.
+
+            Save these configurations to add to your ``user.properties`` file
+
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__.
+
+   .. step:: Gather additional configuration information
+
+      .. procedure::
+
+         .. step:: Find and save your environment ID 
+
+            Navigate to your :guilabel:`Environment` page to find your environment ID.
+
+         .. step:: Find and save your cluster ID
+
+            Navigate to your :guilabel:`Cluster Settings` page to find your cluster ID.
+
+         .. step:: Find and save your cluster Bootstrap server
+
+            In your :guilabel:`Cluster Settings` page, find your cluster Bootstrap 
+            server.
+
+         .. step:: Find and save your cluster REST endpoint
+
+            In your :guilabel:`Cluster Settings` page, find your cluster REST 
+            endpoint.
+
+   .. step:: Add the configuration properties to your ``user.properties`` file
+
+      The location of your ``user.properties`` file varies according to your operating
+      system. For more information, see :ref:`file-location`. 
+
+      Use the following example to add the configuration properties to your 
+      ``user.properties`` file:
+
+      .. code-block::
+
+         spring.profiles.active: confluent-dev
+         migrator.confluent.environment.environment-id: <Environment ID>
+         migrator.confluent.environment.cluster-id: <Cluster ID>
+         migrator.confluent.environment.bootstrap-server: <xx.region.provider.confluent.cloud:9092>
+         migrator.confluent.environment.rest-endpoint: <https://xxx.region.provider.confluent.cloud:443>
+
+         migrator.confluent.cloud-credentials.api-key: <Cloud API key>
+         migrator.confluent.cloud-credentials.api-secret: <Cloud API Secret>
+
+         migrator.confluent.kafka-credentials.api-key: <Cluster API key>
+         migrator.confluent.kafka-credentials.api-secret: <Cluster API Secret>
+
+   .. step:: Restart Relational Migrator 
+
+Next Steps
```

Comment:
[spacing suggestion]

I recommend removing the extra line(s) between bullet points.

Reply:
Done.